storagenode/peer.go

```diff
@@ -261,7 +270,9 @@ func (peer *Peer) Run(ctx context.Context) error {
 		peer.Log.Sugar().Infof("Private server started on %s", peer.PrivateAddr())
 		return ignoreCancel(peer.Server.Run(ctx))
 	})
-
+	group.Go(func() error {
```

Comment:
Move it closer to `peer.Storage2` elements.

Reply:
done